packages/grafana-ui/src/components/Input/Input.tsx

```diff
@@ -25,7 +25,7 @@ export interface Props extends Omit<HTMLProps<HTMLInputElement>, 'prefix' | 'siz
 
 interface StyleDeps {
   theme: GrafanaTheme2;
-  invalid: boolean;
+  invalid?: boolean;
```

Comment:
```suggestion
  isInvalid?: boolean;
```
use `is` for prefix for boolean values

maybe refactor this into being valid instead. I find that negation should be avoided.
What I mean by that is that double negation is hard to understand 🤣 .

if (!invalid) {} <-- is it not not valid?